src/include/mpir_attr.h

```diff
@@ -39,11 +39,11 @@ MPII_Attr_copy_c_proxy(MPI_Comm_copy_attr_function * user_function,
                        MPIR_Attr_type attrib_type, void *attrib, void **attrib_copy, int *flag);
 
 typedef struct copy_function {
-    int (*C_CopyFunction) (int, int, void *, void *, void *, int *);
-    void (*F77_CopyFunction) (MPI_Fint *, MPI_Fint *, MPI_Fint *, MPI_Fint *,
-                              MPI_Fint *, MPI_Fint *, MPI_Fint *);
-    void (*F90_CopyFunction) (MPI_Fint *, MPI_Fint *, MPI_Aint *, MPI_Aint *,
-                              MPI_Aint *, MPI_Fint *, MPI_Fint *);
+    int (*C_CopyFunction)(int, int, void *, void *, void *, int *);
+    void (*F77_CopyFunction)(MPI_Fint *, MPI_Fint *, MPI_Fint *, MPI_Fint *,
```

Comment:
This is where the new version gets right, +1, although I don't really care one way or the other as long as it's consistent.